qubes-gui-client/src/lib.rs

```diff
@@ -117,7 +117,7 @@ impl Client {
     }
 
     /// Creates a daemon instance
-    pub fn daemon(domain: u16, xconf: qubes_gui::XConf) -> io::Result<Self> {
+    pub fn daemon(domain: u16, xconf: qubes_gui::XConfVersion) -> io::Result<Self> {
```

Comment:
Since the implementation discards the `version` part anyway, perhaps it'd be better for this function to only take an `XConf`?